From 4902ca6d9a84afda1e47982be465b773cdf6cc15 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 8 May 2018 10:25:05 +0800
Subject: [PATCH 3734/5242] MLK-18224-1 ARM64: dts: freescale: imx8qm: update
 MU IRQ number

commit  9e89c774c3cb28117fca279f8fc4744f57ef9c92 from
https://source.codeaurora.org/external/imx/linux-imx.git

MU IRQ number is incorrect, update it.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 190a6d2..5607348 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -106,7 +106,7 @@
 	mu: mu@5d1c0000 {
 		compatible = "fsl,imx8-mu";
 		reg = <0x0 0x5d1c0000 0x0 0x10000>;
-		interrupts = <GIC_SPI 176 IRQ_TYPE_LEVEL_HIGH>;
+		interrupts = <GIC_SPI 177 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-parent = <&wu>;
 		fsl,scu_ap_mu_id = <0>;
 		status = "okay";
-- 
1.7.9.5

